domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/accessprofiles/RoleToAccessProfileService.java

```diff
@@ -11,4 +12,6 @@
     List<RoleToAccessProfiles> findByRoleName(String roleName);
 
     List<RoleToAccessProfiles> findByCaseTypeReferences(List<String> caseTypeReferences);
+
+    List<CaseRole> findByCaseTypeId(String id);
```

Comment:
why it has to use CaseRole, can't we use RoleToAccessProfiles

Reply:
As per conversation with Kiran, we aggred that it would be better changing CaseRole for RoleAssignment